python_examples/basic_infection_history_simulation/simulation_flow.py

```diff
@@ -0,0 +1,64 @@
+#
+# Example simulation script with data output
```

Comment:
This is nice! Can you add a record of this to the readme in python_examples with a brief explanation of the purpose of this, and a link to epiOS?

Reply:
Will do